cdap-security/src/main/java/co/cask/cdap/security/server/LDAPAuthenticationHandler.java

```diff
@@ -35,19 +33,14 @@
   private static final List<String> mandatoryConfigurables = ImmutableList.of("debug", "hostname", "port", "userBaseDn",
                                                                               "userRdnAttribute", "userObjectClass");
 
-  private static final List<String> optionalConfigurables = ImmutableList.of("bindDn", "bindPassword", "useLdaps",
+  private static final List<String> optionalConfigurables = ImmutableList.of("bindDn", "useLdaps",
                                                                              "userIdAttribute", "userPasswordAttribute",
                                                                              "roleBaseDn", "roleNameAttribute",
                                                                              "roleMemberAttribute", "roleObjectClass");
-  private static boolean ldapSSLVerifyCertificate = true;
 
-  /**
-   * Create a new Authentication handler to use LDAP for external authentication.
-   */
-  @Inject
-  public LDAPAuthenticationHandler(CConfiguration configuration) throws Exception {
-    super(configuration);
-  }
+  private static final List<String> optionalSecureConfigurables = ImmutableList.of("bindPassword");
```

Comment:
Pushed additional changes that make it so that the params from cconf and sconf are combined into one map.
If there's conflicts, the one in sconf is taken, and a warning is logged.